packages/server/lambdas/alfresco/searchQueryBuilder/luceneQueryBuilder.ts

```diff
@@ -105,6 +110,9 @@ export class LuceneQueryBuilder implements SearchQueryBuilder {
         case SearchParameterName.PARENT:
```

Comment:
Are parent-stuff something that can still be used or is this deprecated stuff now?

Reply:
It's still in use for file listing